### PR TITLE
update documentation 

### PR DESCRIPTION
**Pull request by sohnoun**
update documentation after the credAccount refactor

### CODE REVIEW

**Comment by jbremond** [comment]
Hello,

Thanks for this documentation update.

Regards,
Joris

File: documentation/en/source/pages/templates-spec/builders/builders-flexiant.rst [hunk 1]
```diff
@@ -57,11 +57,11 @@ Used to authenticate the Flexiant platform.
 
 The Flexiant cloud account has the following valid keys:
 
-* ``file`` (optional): a string providing the location of the account information. This can be a pathname (relative or absolute) or an URL.
 * ``name`` (mandatory): a string providing the name of the cloud account. This name can be used in a builder section to reference the rest of the cloud account information.
+* ``apiUsername`` (mandatory): a string providing your API username. To get your api username, log in to flexiant cloud orchestrator, click on Settings > Your API Details
```
Comment (emuus) [inline documentation/en/source/pages/templates-spec/builders/builders-flexiant.rst]: initial caps for Flexiant

File: documentation/en/source/pages/templates-spec/builders/builders-aws.rst [hunk 2]
```diff
@@ -69,15 +69,14 @@ Used to authenticate to AWS.
...
-* ``name`` (mandatory): a string providing the name of the cloud account. This name can be used in a builder section to reference the rest of the cloud account information.
-* ``secretAccessKey`` (mandatory): A string providing you AWS secret access key. To get your secret access key, sign into AWS (aws.amazon.com), click on Security Credentials > Access Credentials > Access Keys. Click on the Show button to reveal your secret key
 * ``type`` (mandatory): a string providing the cloud account type: aws.
+* ``name`` (mandatory): a string providing the name of the cloud account. This name can be used in a builder section to reference the rest of the cloud account information.
+* ``accountNumber`` (mandatory): A string providing your AWS account number. This number can be found at the top right hand side of the Account > Security Credentials page after signing into amazon web services
```
Comment (emuus) [inline documentation/en/source/pages/templates-spec/builders/builders-aws.rst]: initial caps for Amazon Web Services

File: documentation/en/source/pages/templates-spec/builders/builders-flexiant.rst [hunk 3]
```diff
@@ -57,11 +57,11 @@ Used to authenticate the Flexiant platform.
...
 * ``password`` (mandatory): a string providing your flexiant cloud orchestrator account password
-* ``username`` (mandatory): a string providing your API username. To get your api username, log in to flexiant cloud orchestrator, click on Settings > Your API Details
-* ``wsdlURL`` (mandatory): a string providing the wsdl URL of the flexiant cloud orchestrator, for example: https://myapi.example2.com:4442/?wsdl
+* ``wsdlUrl`` (mandatory): a string providing the wsdl URL of the flexiant cloud orchestrator, for example: https://myapi.example2.com:4442/?wsdl
```
Comment (emuus) [inline documentation/en/source/pages/templates-spec/builders/builders-flexiant.rst]: initial caps for Flexiant

**Comment by jbremond** [comment]
Thanks for the modification @sohnoun.
I will merge the PR.

Regards,
Joris

**Pull request merged**